README.md

```diff
@@ -0,0 +1,56 @@
+# Project Name: Web Development Course Project
+
+## Description:
+
+This project is based on html, css and javascript technology to build and design a capstone web development project. The project was implemented with linters which make sure that some specific features are implemented successfully on the project for best user experience.
+
+The purpose of the project is to create a website that has the best user experience. The linters in this project includes:
+"lighthouse" which help to improve the quality of webpage,
+"webhint" is a customizable linting tool that helps you improve your site's accessibility, speed, cross-browser compatibility, and more by checking your code for best practices and common errors and
+"Stylelint" which helps you avoid errors and enforce conventions in your styles.
+The goal here is to master all of the tools and best practices because they will be used more often throughout Microverse projects and most likely in future job as well.
+
+## Built With
+
+Major languages: html, css, javascript
+
+## Getting Started
+
+- The feature branch was first created
+- then linter.yml file was added to the .github/workflow folder in feature branch
+  -the added file was staged using the git command
+- the featured branch was pushed to the remote repository
+- pull request was opened on the remote repository which makes the linters file failed some test
+- then the linter was set up properly by installing its dependencies
+- after the linter was properly set up, the html file was created
+- the whole project was properly set up in the html file
+- after the html file is being set up, the style file was added
+- the necessary styling was added to the style file.
+- added javascript code to alter the mobile menu (DOM manipulation).
+
+## Author
+
+👨 Abimbola Favour Adedeji
+
+GitHub: [@fayob](https://github.com/fayob)
+
+Twitter: [@fabimworld2536](https://twitter.com/Fabimworld2536)
+
+LinkedIn: [@abimbola-ade](https://www.linkedin.com/in/abimbola-ade)
+
+## Live Demo
+
+[my_portfolio](https://fayob.github.io/portfolio-setup-project/)
+
+## Contributing
+
+Contributions, issues, and feature requests are welcome!
+Feel free to check the [issues page](#).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
```

Comment:
- [ ] Kindly add an "Acknowledgements" section or similar that gives appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/).